util/steambox

```diff
@@ -51,9 +51,14 @@ declare -a DRIDEVS=()
 if [[ -d /dev/dri ]] ; then
 	DRIDEVS+=(/dev/dri/*)
 fi
-if [[ -d /dev/ati ]] ; then 
+if [[ -d /dev/ati ]] ; then
 	DRIDEVS+=(/dev/ati/*)
 fi
+for cdev in nvidiactl nvidia0 nvidia-modeset; do
```

Comment:
Pushed anew.  It's a little verbose, but I wanted to preserve the character device check.